IDPro blog post (#3426)
FusionAuth/fusionauth-site: astro/src/content/blog/why-members-idpro.mdx

---
publish_date: 2024-12-16 

> title: Why We Joined IDPro

Reviewer: FusionAuth Renews IDPro Membership: Advancing Identity Education for Developers

> title: Why We Joined IDPro

Reviewer: I'd also consider changing the URL to fusionauth-idpro-membership-benefits

Author: done.

> title: Why We Joined IDPro

Reviewer: ```suggestion
title: FusionAuth Renews IDPro Membership: Advancing Identity Education for Developers
```

> description: What is IDPro and why is FusionAuth a member?

Reviewer: ```suggestion
description: FusionAuth Renews IDPro Membership: Advancing Identity Education for Developers
```

> description: What is IDPro and why is FusionAuth a member?

Reviewer: ```suggestion
description: Learn how FusionAuth's renewed IDPro membership benefits developers. Get information about  identity trends, community engagement, and educational resources that keep our CIAM solution ahead of the status quo.
```

authors: Dan Moore
image: /img/blogs/zero-trust-idps/zero-trust-and-how-idps-factor-into-it-header-image.png
categories: Community
tags: idpro, standards, community
excerpt_separator: "{/* more */}"
---

> FusionAuth recently renewed its membership in [IDPro](https://idpro.org/). IDPro is the organization for identity professionals, similar to how the American Bar Association is the professional association for lawyers in the USA, but with a lot less history and influence.

Reviewer: ```suggestion
In a perfect world, every developer would undrestand OIDC, OAuth, and the intricacies of digital identity. Well, that isn't the world that we live in, but FusionAuth is doing our part to help make it a reality. That's why we've renewed our membership to [IDPro](https://idpro.org/), the professional organization for identity experts. 
```

As a company, FusionAuth has a foundational commitment to educating developers about identity in their own language. This includes [our vendor agnostic articles](/articles/), explaining the concepts of identity and our [free community download](/download), which allows developers to use a full feature OAuth and OIDC server for their applications or to learn about the concepts.

{/* more */}

We also think identity should be as complicated as it needs to be, and *not a bit more*. That's why we use terms engineers know in our documentation and application. Terms like `application` instead of `client` and `identity provider` instead of the jargon of the OAuth or OIDC standards, `Authorization Server` or `OpenID Provider` respectively.

> 

Reviewer: ```suggestion
## Why FusionAuth Supports IDPro
```

Why would FusionAuth support an organization focused on digital identity professionals, not software engineers?

The short answer is that our community depends on us keeping on top of new developments in identity.

The slightly longer answer is that IDPro's community and education efforts help elevate the entire digital identity ecosystem. As a happy side benefit, they help the FusionAuth team keep on top of new developments.

Let's talk a bit more about IDPro and their benefits.

> ## Community

Reviewer: ```suggestion
## Community Engagement
```

IDPro is fostering a community of identity professionals. These folks are excited to be pushing the identity state of the art forward. If you ever want to hear someone wax poetic on the details of policy based access controls or the details of identity proofing, these are your people.

The main way I interact with the community is through the IDPro slack. This has over 1,700 members and dozens of channels covering all aspects of digital identity.

I check in periodically to read discussions on channels focused on:

* CIAM (customer identity and access management)
* identity for developers
* OAuth
* and more

There are other ways to be involved with the IDPro community, including meeting up with fellow members at conferences, watching their videos, or volunteering to write for them.

> ## Education

Reviewer: ```suggestion
## Educational Resources
```

The [IDPro Body of Knowledge (BoK)](https://bok.idpro.org/) documents best practices and concepts across the identity space. From token lifetimes to MFA to access governance, there's a detailed, reviewed article for almost any digital identity related topic. And, as mentioned above, they are [looking for contributions](https://bok.idpro.org/submissions/).

In addition to this free resource, IDPro offers a certification, CIDPro. I personally hold this certification. Pursuing it deepened my understanding of the digital identity landscape.

I was intimidated by the test. I delayed taking it for almost a year after purchasing the voucher. I studied mostly by reading the BoK. The test contents were a mix of software development, identity, and standards knowledge. Studying for it definitely increased my knowledge around topics such as federation, the user lifecycle, and privacy. Here's [the study guide](https://idpro.org/cidpro-reference-resources/) if you want to learn more.

I have encouraged other FusionAuth team members to pursue CIDPro as well.

> ## Filter And Transmit

Reviewer: ```suggestion
## Filtering Identity Trends for Developers
```
